### DISCUSSION TITLE
Board Version Compatibility

### DISCUSSION THREAD

**Discussion by johnsethwatkins** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#36] [Q&A] [answered]
I have a v1.2 board up and running.

I have an EU 220V 3 phase supply in my house. I have read that to properly monitor the supply and my 3 phase equipment I need to stack two boards. Two boards will be useful in any event as my supply is not as good as I would like and it is common to find >20v variations across the phases which I'd like to track independently.

If I now purchase a new board from you it will be v1.x (1.4?) that you will send to me.

Does v1.2 stack with the current version (physically and in terms of components), OR, do I need to order two new boards to complete my set-up.

Thanks
John.

**Comment by CircuitSetup** [comment, marked as answer]
Since you'd be measuring 3 voltages, it would be best to have all of the same version of meter. While it will work, the calibration for the v1.2 board will be much different from the newer v1.4 meters. 

If you want, I can give you a large discount on a new v1.4 main board since you were an early adopter 😊 Please send me a message via the Contact form on the site, and I'll get you a coupon code.

**Comment by johnsethwatkins** [comment]
Many thanks. I have ordered a new board and a new expansion board to go with it.

**Reply by wwk-github** [reply]
I need to measure 3 phases as well (Poland - Europe).
@johnsethwatkins if I may ask - are you happy with the 3-phase setup performed by two new boards?
I still hesitate which solution should I use: IoTaWatt or CircuitSetup.

**Comment by johnsethwatkins** [comment]
I never got it working properly no, but I think that is something to do with me and not the equipment. I have one of the three phases which flips from reading positive to reading negative. I have tried changing the ST clamp and I have tried changing the position on the board it is plugged into. It just randomly flips, and with one of the three phases being wrong it means the whole house figure (instant and accumulated) is also wrong. I gave up trying to troubleshoot it, but I think your message has given me a reason to go back to it.

As a HomeAssistant user I can say that the software part of this (ESPHome) is very easy to set-up and manage locally, that is a big thumbs up!

**Reply by wwk-github** [reply]
Thanks for your answer.
That is interesting.
Maybe your grid-power provider is doing this switches as they see it fit but it is strange that only 1 phase is changing.
